### ISSUE TITLE
Solve TypeScript typings path for SCSS imports

### ISSUE THREAD

**Issue by charliegdev** [charliegdev/waffle#38]
Right now, if I open any `.jsx` files that imports `.scss` files, such as:

- `Task.jsx`
- `Tasks.jsx`

The SCSS import triggers a TypeScript warning:

![scss](https://user-images.githubusercontent.com/8703536/58397520-3a6ea680-8006-11e9-8a33-179a568eb7f9.PNG)

What's funny about this, is that as long as I have another typing file opened in VSCode (`typings/declarations.d.ts`), this error won't show.

`typings/declarations.d.ts` contains a hack to disable this SCSS import warning:

```typescript
declare module "*.scss" {
  const content: any;
  export default content;
}
```

So, it seems VS Code has trouble figuring out the correct location for typing files, and this may be an easy config issue to fix. 

**Timebox to 2 hours.**

**Comment by charliegdev** [comment]
Moving this back to **In Progress** since this is happening again. The previous solution doesn't work, and I'll probably need to generate type files for `.scss`.

**Comment by charliegdev** [comment]
Solved in PR #45
